src/loaders/ccloudResourceLoader.ts

```diff
@@ -500,6 +500,32 @@ export class CCloudResourceLoader extends CachingResourceLoader<
     return resources;
   }
 
+  /**
+   * Get the tables / views / columns of a given Flink database via system catalog queries.
+   *
+   * @param database The Flink database to get the relations and columns for.
+   * @param forceDeepRefresh Whether to bypass the ResourceManager cache and fetch fresh data.
+   * @returns Array of {@link FlinkRelation} objects representing the relations in the cluster.
+   */
+  public async getFlinkRelations(
```

Comment:
Same pattern as `getFlinkUDFs()`: https://github.com/confluentinc/vscode/blob/27ef8b63db796f9efea1168ed4496cd85b3aab13/src/loaders/ccloudResourceLoader.ts#L529-L554
...and the `getFlinkAI*()` methods:
https://github.com/confluentinc/vscode/blob/27ef8b63db796f9efea1168ed4496cd85b3aab13/src/loaders/ccloudResourceLoader.ts#L556-L662